workday/13.0/modules/ROOT/pages/workday-connector-examples.adoc

```diff
@@ -0,0 +1,349 @@
+= Workday Connector 13.0 Examples - Mule 4
+:page-aliases: connectors::workday/workday-to-add-fund-to-service.adoc, connectors::workday/workday-to-create-position.adoc, connectors::workday/workday-connector-examples.adoc
+
+Anypoint Connector for Workday (Workday Connector) provides two financial management services examples that enable you to see how to configure Workday Connector works using Anypoint Studio and  Anypoint Design Center’s Flow Designer:
+
+* <<add-fund,Add a Fund to the Financial Management Service>>
+* <<create-position-request,Create a Position Request>>
+
+== Prerequisites
+
+You must have:
+
+* Access to Workday
+* Your Workday username, password, and tenant name
+
+[[add-fund]]
+== Add a Fund to the Workday Financial Management Service (Studio Example)
+
+This example uses the *Invoke* operation to insert a fund in the Workday financial management service.
+
+== Configure HTTP listener
+
+Configure *HTTP Listener* to initiate a Mule flow when a call is made to the `/` account path on localhost port 8081.
+
+This example uses variables for some field values. You can either:
+
+* Replace the variables with their values in the code.
+* Provide the values for each variable in a properties file and then refer to that file from the connector configuration.
+
+If you don't know how to use a properties file, see xref:mule-runtime::mule-app-properties-to-configure.adoc[Configuring Property Placeholders].
+
+To configure *HTTP Listener*:
+
+. Create a new Mule project in Studio.
+. In the *Mule Palette* view, search for *http* and select the *Listener* operation:
++
+.Select the Listener operation in the Mule Palette view.
```

Comment:
I don't use these for images so I'm not sure what the convention is. Is it supposed to be title case?